CHANGELOG.md

```diff
@@ -13,6 +14,9 @@
 * Core-Addon
   * [#299](https://github.com/mozilla-rally/core-addon/pull/299): Change the link to the sample Rally demo addon.
 
+* Core-Addon
```

Comment:
I think you need to rebase against your other PR (which is fine and I think GH might prompt you to do this). Should be fine but something to keep an eye on :)